core/templates/domain/topic/TopicObjectFactorySpec.ts

```diff
@@ -109,6 +110,28 @@ describe('Topic object factory', () => {
     ]);
   });
 
+  it('should warn user when no skill Ids were added for the diagnostic test',
+    () => {
+      _sampleTopic._skillsForDiagnosticTest = [];
+      expect(_sampleTopic.validate()).toEqual([
+        'Skill for the diagnostic test in the topic should not be empty.'
+      ]);
+    });
+
+  it('should warn user when no skill Ids were added for the diagnostic test',
```

Comment:
Wrong description. Please be careful when writing test descriptions; always ensure that the test description matches what is being tested.

Reply:
Rephrased